solutions/instances/variables.tf

```diff
@@ -273,3 +273,19 @@ variable "scc_workload_protection_access_tags" {
     error_message = "Tags must match the regular expression \"[\\w\\-_\\.]+:[\\w\\-_\\.]+\", see https://cloud.ibm.com/docs/account?topic=account-tag&interface=ui#limits for more details"
   }
 }
+
+########################################################################################################################
+# EN Configuration variables
+########################################################################################################################
+
+variable "scc_en_reply_to_email" {
+  type        = string
+  description = "The email address used when receiving an email from the Event Notifications"
+  default     = "no-reply@ibm.com"
+}
+
+variable "scc_en_email_list" {
+  type        = list(string)
+  description = "The list of emails to send out."
```

Comment:
Needs better description here. Consumer need to know what exactly the emails will be used for